docs/EmbeddedAppIOS.md

```diff
@@ -11,11 +11,26 @@ Since React makes no assumptions about the rest of your technology stack – it
 
 ## Requirements
 
+- Xcode project - If you don't have native app, create a sample Xcode project now. 
 - [CocoaPods](http://cocoapods.org/) – `gem install cocoapods`
 - [Node.js](http://nodejs.org)
   - Install **nvm** with [its setup instructions here](https://github.com/creationix/nvm#installation). Then run `nvm install node && nvm alias default node`, which installs the latest version of Node.js and sets up your terminal so you can run it by typing `node`.  With nvm you can install multiple versions of Node.js and easily switch between them.
+- Create a `package.json` file in your root directory with these contents:
+```
+{
+  "name": "embedded",
+  "version": "1.0.0",
+  "description": "Integrating React Native with Existing Apps",
+  "main": "index.js",
+  "homepage": "https://facebook.github.io/react-native/docs/embedded-app-ios.html",
+  "dependencies": {
+    "react-native": "^0.26.0",
+    "react": "~15.0.2"
+  }
+}
+```
 - Install the `react-native` package from npm by running the following command in the root directory of your project:
-  - `npm install react-native`
```

Comment:
The current React incompatibility is a temporary issue (see https://github.com/facebook/react-native/pull/7881 for details). As @davidyaha says, it would be better to keep these instructions future proof and update the instructions to use `npm install --save`